script.js

```diff
@@ -303,8 +298,23 @@ err.addEventListener('click', () => {
 	err.classList.add('hidden');
 });
 
-const userDetails = JSON.parse(localStorage.getItem('User'));
-if (userDetails) {
-	emailval.value = userDetails.Email;
-	nameholder.value = userDetails.Name;
+form.addEventListener('input', () => {
+	const emailadd = emailval.value;
+	const name = nameholder.value;
+	const message = userMessage.value;
+	const obj = {
+		Name: name,
+		Email: emailadd,
+		Message: message,
+	};
+
+	localStorage.setItem('User', JSON.stringify(obj));
+});
```

Comment:
As mentioned by the previous reviewer, form data is still not getting persisted in the browser local storage. I suggest that you do more refactoring and get the user inputs data to persist 🥲 

![image](https://user-images.githubusercontent.com/4247740/146365488-c556f904-1d3e-495a-8824-927fdd207376.png)

Reply:
You should do a pull request before rejecting the review. What you checked was the live demo which is the main branch. Current changes are deployed to the development branch.  Please be sincere next time you review someone's pull request. It might be their last.

Reply:
This is true! I apologize for my mistake 🙏🏻. 
Next reviewer please take note. Thank you in advance!